### PR TITLE
Doc: Replace cross-document links with attributes

### LINKED ISSUE #58: [DOC]Link to bigquery doc results in 404
Link to **BigQuery Quotas and Limits** (under **Additional Resources**) 404s.

https://cloud.google.com/bigquery/quota

### PR DESCRIPTION
**Pull request by karenzone**
Standardize links to Elastic docs to use shared attributes.
Correct link to Quotas and LImits to close #58
Bump to v.4.1.5

### CODE REVIEW

File: docs/index.asciidoc [hunk 1]
```diff
@@ -67,15 +66,15 @@ https://cloud.google.com/docs/authentication/production[Application Default Cred
...
+  {logstash-ref}/persistent-queues.html[Logstash Persistent Queues].
 * Events will be flushed when <<plugins-{type}s-{plugin}-batch_size>>, <<plugins-{type}s-{plugin}-batch_size_bytes>>, or <<plugins-{type}s-{plugin}-flush_interval_secs>> is met, whatever comes first.
   If you notice a delay in your processing or low throughput, try adjusting those settings.
 
 ===== Additional Resources
 
 * https://cloud.google.com/docs/authentication/production[Application Default Credentials (ADC) Overview]
 * https://cloud.google.com/bigquery/[BigQuery Introduction]
-* https://cloud.google.com/bigquery/quota[BigQuery Quotas and Limits]
+* https://cloud.google.com/bigquery/quotas[BigQuery Quotas and Limits]
```
Comment (jsvd) [inline docs/index.asciidoc:77]: ❤️

**Review by jsvd — APPROVED** [review]
LGTM

**Comment by jsvd** [comment]
@karenzone I'll publish once the PR is merged, as the normal publishing process isn't working on this plugin

**Comment by karenzone** [comment]
@jsvd. Thanks for the review and for publishing. ❤️

**Comment by jsvd** [comment]
Published as 4.1.5 https://rubygems.org/gems/logstash-output-google_bigquery/versions/4.1.5-java

**Pull request merged**